linux/setup_omero_ice35.sh

```diff
@@ -4,7 +4,7 @@ set -e -u -x
 
 source settings.env
 
-SERVER=http://downloads.openmicroscopy.org/omero/5.0.6/artifacts/OMERO.server-5.0.6-ice35-b53.zip
+SERVER=http://downloads.openmicroscopy.org/omero/5.0.8/artifacts/OMERO.server-5.0.8-ice35-b60.zip
```

Comment:
Good question, this is getting towards having stable vs develop in the scripts. Advantage of plain `omero` is the docs will always recommend installing our most recent stable version, OTOH `omero50` is more explicit about what's happening.